lib/admin-page.php

```diff
@@ -449,6 +449,57 @@ function classicpress_check_can_migrate() {
 	// TODO: Add instructions if WP too old.
 	echo "</td></tr>\n";
 
+	// Check: Conflicting Theme
+	$theme = wp_get_theme();
```

Comment:
The parent and the child theme (if any) should both be checked.

Reply:
Updated patch should correct this now.